### PR TITLE
added decorators for the major factions

### PR DESCRIPTION
**Pull request by nenau-again**
As requested in https://github.com/naev/naev/issues/936

The decorators consist in ships and logos of the faction. And I reduced the detection radius as I thought it was more natural.

My main concern is about  the readability. Please tell me if you think I should darken the pictures.

If people agree with the concept, I will add smaller ones for interesting places (goddard, krain factory, piracy spots, traderoutes, nebulae...)
For collective, I'm unsure : is it possible to remove the picture with an unidiff once the faction has been beaten ?

### CODE REVIEW

**Comment by bobbens** [comment]
Thanks! I looked at it and have a few comments. First, it might be better to make the decorators more simple and larger to compensate, so that when you zoom in you don't notice them too much, only having them appear when scaled out (we could also just hide them when zooming in). Second, I have a feeling they are too bright. Might be worth darkening them (or making them 50% transparent).

> If people agree with the concept, I will add smaller ones for interesting places (goddard, krain factory, piracy spots, traderoutes, nebulae...)

That would be cool, however, I'm not sure if all the small things would necessarily need a decorator. In particular, I think the nebulae would be most important to have a really nice decorator for.

> For collective, I'm unsure : is it possible to remove the picture with an unidiff once the faction has been beaten ?

It should be possible to add this functionality, but given that they are a few systems, I'm not sure to what point it warrants it's own decorator.

Basically one of the design questions I think that is still open is what function these decorators should be. If it is just showing the macro-layout of the universe, then major factions+nebulae should probably be enough, but if it is just some fancier eye-candy, it can be done at lower levels. However, this detailed eye-candy might also be implementable using shaders or something so that when you zoom in you can see if a system has asteroids and stuff by showing little moving things. Thoughts?

**Comment by nenau-again** [comment]
New version of map decorators. I did not tackle the issue of nebula decorator since I believe this one is more subtile (as there is no obvious logo nor ship (except for the ghost ships, but it would spoil to show how they look on the map)).
I don't really understand the idea that decorators should show asteroids as asteroids are per-system stuff. There is no such thing as multi-system asteroid field.
But maybe showing little moving stuff could be cool some way. However I am pretty sure there is nothing in space that moves fast enough to be considered as moving at such a great scale. Maybe flares ?

**Comment by bobbens** [comment]
Looks much better, thanks!
